### PR TITLE
HAR support

### PR DESCRIPTION
**Pull request by snshn**

### CODE REVIEW

**Review by awtkns — APPROVED** [review]
@KhoomeiK would you be able to take a look at this aswell?

File: bananalyzer/data/generate_examples.py [hunk 1]
```diff
@@ -101,6 +120,67 @@ async def llm_annotate_example(
...
+        page = await context.new_page()
+
+        # Generate random example ID and create new directory
+        example_id = str(uuid.uuid4())
+        example_dir_path = f"./static/{example_id}"
+        os.makedirs(example_dir_path, exist_ok=True)
+
+        # Make browser record all network responses into a HAR file
+        now = datetime.now()
+        timestamped_har_path = os.path.abspath(f"./static/{example_id}/cache/bananas." + now.strftime("%s") + ".har")
```
Comment (awtkns) [inline bananalyzer/data/generate_examples.py:136]: typically i perfer pathlib.Path but this works too

**Comment by awtkns** [comment]
Just make sure you format the files :)

File: bananalyzer/data/generate_examples.py (hunk 1 above)
Comment (snshn) [reply bananalyzer/data/generate_examples.py:136]: I'll switch it to be consistent with the rest of the code, thank you for the tip Adam!

**Pull request closed without merging**